ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/router-bgp-evpn-mpls.yml

```diff
@@ -37,6 +37,10 @@ router_bgp:
     - ip_address: 192.168.255.2
       peer_group: EVPN-OVERLAY-PEERS
   address_family_evpn:
+    next_hop_mpls_resolution_ribs:
```

Comment:
We support three levels here
```eos
router bgp 1
   address-family evpn
      next-hop mpls resolution ribs tunnel-rib test tunnel-rib colored system-colored-tunnel-rib system-connected
```

So I suggest primary_rib, secondary_rib, tertiary_rib.

Make make each of them a dict with `type: <str; system-connected | tunnel-rib | tunnel-rib-colored` and `rib_name`.
For now the name when type is tunnel-rib-colored can be hardcoded to the sytem one since that is the only available option.

Reply:
Alternative have a key per option with `priority` 1-3 (or unset)

Reply:
Updated to use a list.. but still need to figure out the priority.